t/t5325-commit-graph-bloom.sh

```diff
@@ -0,0 +1,255 @@
+#!/bin/sh
+
+test_description='commit graph with bloom filters'
```

Comment:
It would be better if this test script grew with your changes. Your commit adding the `--bloom` option could be the start.

Don't worry about it right now, but mention in the commit message that you plan to split this test across the earlier commits.